docs/README.md

```diff
@@ -74,26 +74,28 @@ is installed. `Succeeded` should appear in the **Status** column.
 
 1. Create the `Subscription` object with the desired `spec.channel`.
 
-    ```yaml
-    apiVersion: operators.coreos.com/v1alpha1
-    kind: Subscription
-    metadata:
-      name: sailoperator
-      namespace: openshift-operators
-    spec:
-      channel: "0.1-nightly"
-      installPlanApproval: Automatic
-      name: sailoperator
-      source: community-operators
-      sourceNamespace: openshift-marketplace
-    ```
+   ```bash
+   kubectl apply -f - <<EOF
+       apiVersion: operators.coreos.com/v1alpha1
+       kind: Subscription
+       metadata:
+         name: sailoperator
+         namespace: openshift-operators
+       spec:
+         channel: "3.0-nightly"
```

Comment:
Is this OK? I noticed today that the last build in `3.0-nightly` is from May 13. The newer builds are in the `0.1-nightly` channel.

Reply:
Ohh, i thought `3.0-nightly` would have the most latest ones. I'll update it.